content/en/docs/contribute/participate/roles-and-responsibilties.md

```diff
@@ -188,4 +188,10 @@ To apply:
 
 2. Assign the PR to one or more current SIG Docs approvers.
 
-If approved, a SIG Docs lead adds you to the appropriate GitHub team. Once added, [K8s-ci-robot](https://github.com/kubernetes/test-infra/tree/master/prow#bots-home) assigns and suggests you as a reviewer on new pull requests.
\ No newline at end of file
+If approved, a SIG Docs lead adds you to the appropriate GitHub team. Once added, [K8s-ci-robot](https://github.com/kubernetes/test-infra/tree/master/prow#bots-home) assigns and suggests you as a reviewer on new pull requests.
+
+## {{% heading "whatsnext" %}}
+
+Learn about specialized roles in SIG Docs:
+
+- [PR wranglers](/docs/contribute/participating/pr-wranglers)
```

Comment:
The _role_ is approver, I think. All approvers become PR wranglers on a rota.

Reply:
@sftim addressed this in the latest commit.